### DISCUSSION TITLE
I unable to get the coverage report for VS2019 And Fine coverage report 1.1.10

### DISCUSSION THREAD

**Discussion by ajaygandhi-avalara** [FortuneN/FineCodeCoverage#181] [General]
I am unable to get the coverage report for VS2019 And the Fine coverage report 1.1.10.
My window is always blank 
![image](https://user-images.githubusercontent.com/16475253/130342867-7f15a53e-911b-4006-a202-7101e06ac600.png)

And the test is here 
![image](https://user-images.githubusercontent.com/16475253/130343343-86e64ef7-df63-41ec-8ec9-5bc003c0b43e.png)

and the setting is as below 
![image](https://user-images.githubusercontent.com/16475253/130343350-7bd7f168-5520-40f1-a76f-cc9e481708f7.png)

Anything I am missing?

**Comment by tonyhallett** [comment]
Can you post the full contents of the FCC Output window pane.
![image](https://user-images.githubusercontent.com/11292998/130348862-2ac6b289-7951-46a5-9a60-8dee94eef34a.png)

Are you using version 1.1.10.144 taken from Release v1.1.144 ?

**Comment by tonyhallett** [comment]
> Prerequisites
Only that the test adapters are nuget packages. For instance, the NUnit Test Adapter extension is not sufficient.

FCC copies dlls to a sub folder so that may cause issues with paths.

Is there any reason why you have both CoverletConsoleGlobal and CoverletConsoleLocal ?

**Comment by tonyhallett** [comment]
@ajaygandhi-avalara Hav eyou resolved your issue ?
